src/static/riot/competitions/detail/_header.tag

```diff
@@ -40,6 +40,10 @@
                                 <span class="detail-label">Current phase ends:</span>
                                 <span class="detail-item">{get_end_date(competition)}</span>
                             </div>
+                            <div>
+                                <span class="detail-label">Current server time:</span>
+                                <span class="detail-item" id="server_time">{pretty_date(CURRENT_DATE_TIME)}</span>
```

Comment:
where did `pretty_date` come from? cool

Reply:
This was a already created function in `ours/utils.js`